lib/scrabble.rb

```diff
@@ -1,4 +1,60 @@
 module Scrabble
   # require your gems and classes here
-  # require_relative 'scrabble/whatevs'
+	#require_relative 'scrabble/whatevs'
+	class Foo
+		def convert(letter_value)
+			return "" if letter_value == 0
+		end
+
+		LETTER_VALUE = {"A"=>1, "E"=>1, "I"=>1, "O"=>1, "U"=>1, "L"=>1, "N"=>1, "R"=>1, "S"=>1, "T"=>1, "D"=>2, "G"=>2, "B"=>3,"C"=>3, "M"=>3, "P"=>3, "F"=>4, "H"=>4, "V"=>4, "W"=>4,  "Y"=>4,"K"=>5, "J"=>8, "X"=>8, "Q"=>10, "Z"=>10}
+
+
+		# def self.score(word)
+		# 	x = word.split(//)
+		# 	# puts x
+		# 	x.each do |value|
+		# 		value = value.upcase
+		# 		value = value.to_sym
+		# 		# puts LETTER_VALUE[value]
+		# 		word = LETTER_VALUE[value]
+		# 		word = sum{}
+		# 		puts word
+		# 	end
+		# end
+		
+		def self.score(word)
+			sum = 0
+			word = word.upcase
+			word = word.split(//).each do |value|
+				sum = sum+LETTER_VALUE[value]
+			end
+			return sum
+		end
+
+		def self.highest_score_from(array_of_words)
+			# answer = array_of_words
+			# answer = array_of_words.max_by{|word| score(word)} 	#works but limits effective 														application of conditionals
+			answer = array_of_words.sort_by!{|word| score(word)}
```

Comment:
In combination with using ```#pop``` below, since we are operating on the original copy of ```array_of_words```, one word will be removed from the array each time this method is called (which would be unexpected for code that calls this method).